packages/shared/src/components/modals/ConfirmationModal.module.css

```diff
@@ -7,7 +7,7 @@
 
   & :global(.modal) {
     max-width: 23.25rem;
-    padding: 1.5rem 2.5rem;
+    padding: 2rem;
```

Comment:
I think the x-axis padding should still be as below. Also, one thing to note here, is it would affect existing implementations, so kindly check if we should separate the selector for this part.
 
```suggestion
    padding: 2rem 2.5rem;
```

Reply:
The alround is 2rem in the design? 
But will look at extracting it for this specific one

Reply:
I think it should be `2.5rem` for x-axis and `2rem` for y-axis.

![image](https://user-images.githubusercontent.com/13744167/140011424-49c8cd38-d0d8-4479-af81-5c0b0f7173f9.png)